content/elections/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "OHBM OSSIR 2021 elections"
+subtitle: "Make a difference: join the OS SIG!"
+date: 2020-06-30
+draft: false
+sidebar: false # true or false to display the sidebar
+sidebarlogo: fresh-white-alt # From (static/images/logo/)
+include_footer: true
+---
+
+# Make a difference: join the OS SIG!
+
+We are excited to announce this year's Nominations for the OHBM open-science Special Interest Group (OS SIG) for OHBM 2021!
```

Comment:
We've chatted a bit on the Mattermost channel about differentiating between the OSSIG (the whole community) and the OSSIG committee/officers (those official leadership positions). If we want to move forward with that, I might add "committee" or "officers" after `(OS SIG)` here.

Reply:
I like that! I think this confusion has been going on for too long ;)